src/CrashlogFile.ts

```diff
@@ -79,6 +104,107 @@ export class CrashlogFile {
             const line = lines[i];
             this.findPkgPaths(line, i);
         }
+
+        this.parseCrashes();
+    }
+
+    private parseCrashes() {
+        this.crashes = [];
+        let contents = this.fileContents;
+
+        // TODO: redundant code from StandardReporter, how to refactor?
+        // walk the references backwards and replace the file contents
+        for (const ref of [...this.references].reverse()) {
```

Comment:
Why is this loop here?

Reply:
@TwitchBronBron I saw that `CrashlogFile.contents` does not include the mapped source locations generated after `parse()`. This replacement is done in `StandardReported.getFileContents()`. But I want the replacement done before I generate the crashes structure. So the for loop is adapted from `StandardReported.getFileContents()`. But now the code is redundant. What should I do?